lib/off_broadway/kafka.ex

```diff
@@ -1,17 +1,132 @@
 defmodule OffBroadway.Kafka do
-  @callback broadway_config(keyword(), String.t(), integer()) :: keyword()
+  @moduledoc ~S"""
```

Comment:
What does `~S` do?

Reply:
It does but it’s just the way to force it to stop interpolating code inside of a double-quote/curly brace structure